README.md

```diff
@@ -46,25 +46,33 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
-* Pull and start the docker images
+1. Install Docker Community Edition and docker-compose
```

Comment:
The product is _Docker Compose_, the command is `docker-compose`. If you mean the former, switch; if you mean the latter, put within backticks.